stable/wordpress/values.yaml

```diff
@@ -181,28 +181,28 @@ ingress:
     ## A side effect of this will be that the backend wordpress service will be connected at port 443
     tls: false
 
+    ## Set this to true in order to add the corresponding annontations for cert-manager
```

Comment:
add these changes to the values-production.yaml too

Reply:
Done!